content/en/docs/setup/upgrade/istioctl-upgrade/index.md

```diff
@@ -1,27 +1,25 @@
 ---
-title: Upgrade Istio using istioctl [Experimental]
+title: Upgrade Istio using istioctl
 description: Upgrade or downgrade Istio using the istioctl upgrade command.
 weight: 25
 keywords: [kubernetes,upgrading]
 ---
 
-{{< boilerplate experimental-feature-warning >}}
-
-The `istioctl experimental upgrade` command performs an upgrade of Istio. Before performing
+The `istioctl upgrade` command performs an upgrade of Istio. Before performing
 the upgrade, it checks that the Istio installation meets the upgrade eligibility
 criteria. Also, it alerts the user if it detects any changes in the profile
 default values between Istio versions.
 
 The upgrade command can also perform a downgrade of Istio.
 
-See the [`istioctl` upgrade reference](/docs/reference/commands/istioctl/#istioctl-experimental-upgrade)
-for all the options provided by the `istioctl experimental upgrade` command.
+See the [`istioctl` upgrade reference](/docs/reference/commands/istioctl/#istioctl-upgrade)
+for all the options provided by the `istioctl upgrade` command.
 
 ## Upgrade prerequisites
 
 Ensure you meet these requirements before starting the upgrade process:
 
-* Istio version 1.3.3 or higher is installed.
+* Istio version 1.4.4 or higher is installed.
```

Comment:
is this pre-req right? I don't recall upgrade being promoted from `istioctl experimental upgrade` in 1.4 series.

Reply:
I expanded the context and noticed this is are prereqs for the original installed version. Not sure how to clarify, but should be clarified if possible

Reply:
sorry, can you explain more, or should we add something like "Please upgrade your cluster to 1.4.4".